images/linux/scripts/installers/rust.sh

```diff
@@ -19,19 +19,13 @@ source $CARGO_HOME/env
 rustup component add rustfmt clippy
 cargo install --locked bindgen cbindgen cargo-audit cargo-outdated
 
-# Permissions
-chmod -R 777 $(dirname $RUSTUP_HOME)
-
 # Cleanup Cargo cache
 rm -rf ${CARGO_HOME}/registry/*
 
 # Update /etc/environemnt
-prependEtcEnvironmentPath "${CARGO_HOME}/bin"
+prependEtcEnvironmentPath "${HOME}/.cargo/bin"
 
-# Rust Symlinks are added to a default profile /etc/skel
-pushd /etc/skel
-ln -sf $RUSTUP_HOME .rustup
-ln -sf $CARGO_HOME .cargo
-popd
+# unset back to default
+unset RUSTUP_HOME CARGO_HOME
```

Comment:
No need to unset it. These variables will unset automatically after script execution.